web_responsive/static/src/css/web_responsive.scss

```diff
@@ -202,12 +248,19 @@ $chatter_zone_width: 35%;
             height: 100%;
 
             .search-input {
-                height: 3em;
+                height: 100%;
             }
 
             .search-results {
-                height: calc(100% - 3em);
+                max-height: calc(100vh - 47px - 4em);
+                overflow-y: hidden;
+                overflow-x: scroll;
                 overflow: auto;
+                background: url("../img/home-menu-bg-overlay.svg"),
+                    linear-gradient(to bottom, #f9f9f9, #ffffff);
+                background-position: center;
+                background-size: cover;
+                border-radius: $border-radius;
```

Comment:
You can remove this styling by adding a new class as explained in other comment below:

```suggestion
```

Reply:
Isn't it better to leave it in the scss to make it easier to overwrite for example in a theme?

Reply:
Something I found out when developing https://github.com/OCA/openupgradelib/pull/148 (utility to migrate from Bootstrap 3 (odoo 11-) to 4 (odoo 12+)) is that it is much easier to understand how an element is styled when you use helper classes.

When using custom css, it's much harder to know if a selector still matches an element, and if the property it is defining is stil needed or is being used fine. You need to do those checks manually always (I have migrated customer themes too), or you might end up having a lot of dead code that increases the browser payload and decreases performance, and having to refactor every now and then (as I actually did with `web_responsive` in https://github.com/OCA/web/pull/1066. With classes, it can be automated.

And I guess that some day we will have Bootstrap 5, so... well, I recommend to use classes where possible now.

---

If Odoo were not using a CSS framework, then yes. All would be easier in plain CSS. But it uses Bootstrap, so it's better to use it wherever possible.